lib/Doctrine/ORM/Cache/NullCacheItemPool.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\ORM\Cache;
+
+use Closure;
+use Doctrine\Common\Cache\Psr6\CacheItem;
```

Comment:
This package is deprecated, you shouldn't need to use it.

Reply:
I'm not sure what I should use instead as I still need an implementation of `Psr\Cache\CacheItemInterface` since getItem must always return one.

Reply:
Maybe you can write a dummy implementation?